### PR TITLE
Feature/email

### PR DESCRIPTION
**Pull request by GaelTikeng**
I made the email link to redirect to an email adress

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -228,8 +228,8 @@ <h1>Opening hours</h1>
         <div class="footer-right">
             <h1>Get in Touch</h1>
             <p><i class="fa fa-map-marker"></i>Location: Total melen, Yaoundé</p>
-            <p><i class="fa fa-paper-plane"></i>Email: Afrostudio@gmail.com</p>
-            <p></p><i class="fa fa-phone"></i>Phone: +237690920054</p>
+            <p><a href="mailto:Gaelinho@yahoo.fr" target="'_blank"><i class="fa fa-paper-plane"></i>Email: Afrostudio@gmail.com</a></p>
+            <p><a href="tel:6909254"><i class="fa fa-phone"></i>Phone: +237690920054</a></p>
```
Comment (nsamedaisy) [inline index.html]: is it safe to expose your contact on  the website ?

**Review by nsamedaisy — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -228,8 +228,8 @@ <h1>Opening hours</h1>
         <div class="footer-right">
             <h1>Get in Touch</h1>
             <p><i class="fa fa-map-marker"></i>Location: Total melen, Yaoundé</p>
-            <p><i class="fa fa-paper-plane"></i>Email: Afrostudio@gmail.com</p>
-            <p></p><i class="fa fa-phone"></i>Phone: +237690920054</p>
+            <p><a href="mailto:Gaelinho@yahoo.fr" target="'_blank"><i class="fa fa-paper-plane"></i>Email: Afrostudio@gmail.com</a></p>
+            <p><a href="tel:6702028854"><i class="fa fa-phone"></i>Phone: +237690920054</a></p>
```
Comment (nsamedaisy) [inline index.html]: number not correct

File: index.html (hunk 1 above)
Comment (GaelTikeng) [reply index.html]: yes i think it is safe

File: index.html (hunk 2 above)
Comment (nsamedaisy) [reply index.html]: is it?

File: index.html [hunk 3]
```diff
@@ -228,8 +228,8 @@ <h1>Opening hours</h1>
         <div class="footer-right">
             <h1>Get in Touch</h1>
             <p><i class="fa fa-map-marker"></i>Location: Total melen, Yaoundé</p>
-            <p><i class="fa fa-paper-plane"></i>Email: Afrostudio@gmail.com</p>
-            <p></p><i class="fa fa-phone"></i>Phone: +237690920054</p>
+            <p><a href="mailto:Gaelinho@yahoo.fr" target="'_blank"><i class="fa fa-paper-plane"></i>Email: Afrostudio@gmail.com</a></p>
+            <p><a href="tel:672028854"><i class="fa fa-phone"></i>Phone: +237690920054</a></p>
```
Comment (nsamedaisy) [inline index.html:232]: cool

**Review by nsamedaisy — APPROVED** [review]
good job

File: index.html (hunk 3 above)
Comment (GaelTikeng) [reply index.html:232]: thank you

**Pull request merged**